_include-media-hidden-classes.scss

```diff
@@ -1,9 +1,14 @@
 $im-num-breakpoints: length($breakpoints);
+$im-hidden-important: false !default;
 $im-hidden-class: "hidden" !default;
 
 @mixin im-hidden($type, $breakpoint) {
   .#{$im-hidden-class}--#{$type}-#{$breakpoint} {
-    display: none;
+    @if $im-hidden-important == false {
+      display: none;
+    } @else {
+      display: none!important;
```

Comment:
Would you mind adding a space before `!important`? Other than that, looks great!